src/components/BarSeries.tsx

```diff
@@ -100,7 +100,7 @@ function BarSeriesRender({
       {data.map(({ x, y }) => (
         <line
           style={style}
-          key={`${x}-${y}`}
+          key={`${xScale(x)}-${yScale(y)}`}
```

Comment:
This change does not seem related to this PR. Does it fix any sort of bug?

Reply:
I think it's wrong to use x and y as the key. We can replace them with the array index as there is nothing that clearly identifies each data point.

Reply:
i used index